lib-release.js

```diff
@@ -143,6 +143,27 @@ promiseSeries([
 
         `));
     }),
+  () => execaSeries([
+    `git checkout ${DEVELOP_TRUNK}`,
+    'git pull',
+    'git commit --allow-empty -m "Trigger real permission checks"',
```

Comment:
@LucaMele 
This would leave empty commits in the history.
We clean this up, though only if we have **force push** privileges.

Reply:
ok for the moment i guess